### PR TITLE
fix: make default import

### PR DESCRIPTION
**Pull request by jquense**
cc @TrySound

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -2,9 +2,9 @@ import React from 'react'
 import invariant from 'invariant'
 import * as Utils from './utils'
 
-export { default as useUncontrolled } from './hook.js';
+export { default as useUncontrolled } from './hook.js'
 
-export function uncontrollable(
+export default function uncontrollable(
```
Comment (TrySound) [inline src/index.js]: Mixing default and named exports may lead to unexpected bugs for dependent packages without esm support (like react-overlays I guess). I already shout in my foot in material-ui with this.

Comment (jquense) [reply src/index.js]: b/c the compiled output is different for the `require` case you mean?

Comment (TrySound) [reply src/index.js]: Yes, webpack is buggy here. The result is compiled into
```js
module.exports = {
  default: {
    default: function () {}
  }
}
```
So babel interop is not able to solve this.

**Comment by TrySound** [comment]
Could you just republish 6.1.0 as 6.2.2?

File: src/index.js (hunk 1 above)
Comment (jquense) [reply src/index.js]: i just added different index's for each case that should handle it I believe

**Pull request merged**